javascriptv3/example_code/cross-services/ddb-item-tracker/README.md

```diff
@@ -0,0 +1,729 @@
+
+#### 4. Verify to and from email address on Amazon SES
+This example requires a minimum of two verified email address.
+
+To verify and email address:
+
+1. Open [AWS SES in the AWS Management Console](https://aws.amazon.com/SES/), and open the **Email Addresses** page.
+2. Choose **Verify a New Email Address**.
+3. Enter a working email address, and choose **Verify This Email Address**.
+4. Open the email in your email application, and verify it.
+
+
+## <a name="runApp"></a>Run the app
+
+1. Clone the [AWS Code Examples Repository](https://github.com/awsdocs/aws-doc-sdk-examples) to your local environment.
+   See [the Github documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) for
+   instructions.
+
+2. Install the dependencies listed in the [root]/javascriptv3/example_code/cross-services/ddb-item-tracker/package.json.
+
+**Note**: These include both AWS SDK for JavaScript v3 client modules and third-party Node.js modules.
+```
+npm install node -g 
+cd javascriptv3/example_code/cross-services/ddb-item-tracker
+npm install
+```
+
+3. Open **items.js**, and replace the following:
+   1. **TABLE_NAME** with the name of the table 
+   2. **BUCKET_NAME** with the name of the Amazon S3 bucket
+   3. **SENDER_EMAIL_ADDRESS** with the address of the email sender. This address must first be verified on Amazon SES.
+4. Start the app.
+```
+node app.js
+```
+The app is now running at [http://localhost:3000/index.html](http://localhost:3000).
+
+
+## <a name="projectFiles"></a>About the project files
+
+The following figure shows you the structure of the project.
+
+![AWS Tracking Application](images/item_tracker_projects.png)
+
+- **images**: contains images used by this tutorial. You can safely delete it from your project.
+- **public folder**: contains most of the code for the app. See [Folder in the public folder](#public) for more details.
+- **app.js**: the main file in the app. It interacts with the Express web framework to execute code. See [app.js file](#appjs) for more details.
+- **package.json**: defines functional attributes of a project that npm uses to install dependencies.
+- **README.md**: the page you're reading.
+- **setup.yaml**: the Amazon Cloud Formation script used to create the resources for this project. See [Create the resources](#createTheResources) for more details.
+
+### <a name="public"></a>Folders in the **public** folder
+
+#### <a name="css"></a>'css' folder
+Contains the CSS style files for the app.
+#### <a name="dynamoAppHelperFiles"></a>'dynamoAppHelperFiles' folder
+Contains script that creates a table using the AWS SDK for JavaScript (v3).
+
+To create the table for the app:
+1. Update the **dynamoClient.js** as described in [libs](#libs)
+2. Update the client files in the **libs** folder as described in ### Folders in the **public** folder
```

Comment:
We're in the "Folders in the public folder" section. Cut and paste error?

Reply:
done